doc/source/install/index.rst

```diff
@@ -2,34 +2,330 @@
+
+.. code-block:: bash
+
+    $ python3 -m venv venv
     $ source venv/bin/activate
-    $ pip install otcextensions
-    $ python setup.py install
+    (venv) $
+
+Now, install all libraries and programs at once with the Python package manager pip:
+
+.. code-block:: bash
+
+    $ pip install otcextensions openstackclient
+
+
+Installation under CentOS
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After a new installation: a C compiler, Python3 with developer libraries, package manager, and virtual environment are required.
+If you want to use python3 which is recommended, you need to install the epel repository, first:
+
+
+.. code-block:: bash
+
+   $ sudo yum update 
+   $ sudo yum install epel-release
+
+No the python packages are needed:
+
+.. code-block:: bash
+
+   $ sudo yum update
+   $ sudo yum install gcc python36 python36-devel python-pip python-virtualenv openssl-devel
+
+A virtual environment seperates your installed packages from other libraries and should be used as well. You can name the virtual environment on your own desires, in our example it is: "venv". The second command will switch on "venv":
+
+.. code-block:: bash
+
+   $ python3 -m venv venv
+   $ source venv/bin/activate
+
+Now, install all libraries and programs at once with the Python package manager pip:
+
+.. code-block:: bash
+
+   $ pip install otcextensions openstackclient
+
+Installation under Fedora (under review)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After a new installation: a C compiler, Python3 with developer libraries, package manager, and a virtual environment are required:
+
+
+.. code-block:: bash
+
+   $ sudo dnf upgrade
+   $ sudo dnf install gcc python3 python3-devel python3-pip python3-virtualenv openssl-devel
+
+The virtual environment will be created and activated. You can name the virtual environment on your own desires, in our example it is "venv":
+
+.. code-block:: bash
+
+   $ python3 -m venv venv
+   $ source venv/bin/activate
+
+Now, install all libraries and programs at once with the Python package manager pip:
+
+.. code-block:: bash
+
+   (venv) $ pip install otcextensions openstackclient
+   
+
+Installation from Github sources
+--------------------------------
+
+The latest state of the packages can be installed with the following approach.
+
+Cloning the Github repository:
+
+.. code-block:: bash
+
+   $ git clone git@github.com:OpenTelekomCloud/python-otcextensions.git
+
+A virtual environment seperates your installed packages from other libraries and should be used as well. You can name the virtual environment on your own desires, in our example it is: "venv". The second command will switch on "venv":
+
+.. code-block:: bash
+
+   $ python3 -m venv venv
+   $ source venv/bin/activate
+   (venv) $
+
+Switch into the new folder which is created by cloning the repository and install install the project dependencies into the virtual environment:
+
+.. code-block:: bash
+
+   (venv) $ cd ./python-otcextensions.git
```

Comment:
.git suffix should not be there. The clone should remove it

Reply:
deleted